frontend/packages/api_client/src/models/ToggleStaticContentPages.ts

```diff
@@ -0,0 +1,65 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * OpenCraft Instance Manager
+ * API for OpenCraft Instance Manager
+ *
+ * The version of the OpenAPI document: api
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { exists, mapValues } from '../runtime';
+/**
+ * 
+ * @export
+ * @interface ToggleStaticContentPages
+ */
+export interface ToggleStaticContentPages {
+    /**
+     * 
+     * @type {string}
+     * @memberof ToggleStaticContentPages
+     */
+    pageName: string;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof ToggleStaticContentPages
+     */
+    enabled: boolean;
+}
+
+export function ToggleStaticContentPagesFromJSON(json: any): ToggleStaticContentPages {
+    return ToggleStaticContentPagesFromJSONTyped(json, false);
+}
+
+export function ToggleStaticContentPagesFromJSONTyped(json: any, ignoreDiscriminator: boolean): ToggleStaticContentPages {
+    if ((json === undefined) || (json === null)) {
+        return json;
+    }
+    return {
+        
+        'pageName': json['page_name'],
+        'enabled': json['enabled'],
+    };
```

Comment:
Same as previous comment, this also would need to be replaced with a function to avoid repetition.